contrib/test/travis/check_all_examples_use_standard_init.sh

```diff
@@ -1,16 +1,16 @@
 #!/bin/bash
-FBASE_SNIPPET=test/travis/firebase_init.cc.snip
-FBASE_NOAUTH_SNIPPET=test/travis/no_firebase_init.cc.snip
-FBASE_NOCONFIG_SNIPPET=test/travis/no_config_init.cc.snip
-for example in `find examples/ -name *.ino`;
+FBASE_SNIPPET=contrib/test/travis/firebase_init.cc.snip
```

Comment:
IIRC I moved all the tests stuff to contrib/ for now, since it was pretty tangled.

So that we can easily package the Arduino library as: root - everything in contrib until we find a better organization/packaging option.